libs/sdk-flutter/android/src/main/kotlin/flutter_secured_storage/RSACipher18Implementation.kt

```diff
@@ -0,0 +1,205 @@
+package flutter_secured_storage
```

Comment:
@dangeross why do we need the keychain specific code in the flutter plugin? Isn't this part of the connect method that the user should implement?

Reply:
I remember these being removed, could be related to the latest rebase.

Reply:
You can see `readSecuredValue` from the flutter_secured_storage package being used here in the getConnectRequest:
```
import flutter_secured_storage.FlutterSecuredStorageHelper.Companion.readSecuredValue
...

class ExampleForegroundService : ForegroundService() {
    ...

    override fun getConnectRequest(): ConnectRequest? {
        val apiKey = applicationContext.getString(R.string.breezApiKey)
        val glNodeConf = GreenlightNodeConfig(null, null)
        val nodeConf = NodeConfig.Greenlight(glNodeConf)
        val config = defaultConfig(EnvironmentType.PRODUCTION, apiKey, nodeConf)

        config.workingDir = PathUtils.getDataDirectory(applicationContext)

        return readSecuredValue(
            applicationContext,
            "${ELEMENT_PREFERENCES_KEY_PREFIX}_${ACCOUNT_MNEMONIC}"
        )
            ?.let { mnemonic ->
                ConnectRequest(config, mnemonicToSeed(mnemonic))
            }
    }
}
```

Reply:
The question is if we should have it there or it should be on the app side.
This code does assume a specific implementation for flutter secure storage.